src/components/core/PivotTable.tsx

```diff
@@ -422,6 +442,8 @@ interface ISortedByColumnIndexes {
     all: number[];
 }
 
+const DEFAULT_ROW_HEIGHT = 28;
```

Comment:
What would be the benefit? I would like to keep the constant close to the pivot table as it's not used anywhere else.

Reply:
I think that pivotTable.tsx is huuge file and we should split it. And small things like constants will pile up in long run

Reply:
We usually keep constants used strictly by given component within the same file. Therefore I would keep it that way. In case the number of constants would rise above the "limit", we can move it to separate file then.